src/password.rs

```diff
@@ -0,0 +1,56 @@
+use std::{error::Error, fmt, str::FromStr};
+
+use serde::{Deserialize, Serialize};
+use zeroize::Zeroize;
+
+use crate::Hidden;
+
+/// A hidden string that implements [`Zeroize`].
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
+#[serde(transparent)]
+pub struct SafePassword {
+    password: Hidden<String>,
```

Comment:
Should this not be a slice so that reallocations can't happen?

Reply:
Good point :+1: Done